static/app/gettingStartedDocs/native/native.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {StepTitle} from 'sentry/components/onboarding/gettingStartedDoc/step';
+
+import {GettingStartedWithNative, steps} from './native';
+
+describe('GettingStartedWithNative', function () {
+  it('all products are selected', function () {
```

Comment:
This does not seem to be the correct test title - perhaps the wrong copy-paste?

Reply:
good catch! yes it was copy and paste 😄 will fix it in a follow-up